cucumber-jparest/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
       <spring.version>6.0.4</spring.version>
+      <maven.deploy.skip>false</maven.deploy.skip>
```

Comment:
Wed don't want to skip this do we

Reply:
I have set the parent pom to be skip true.  Which means parent and all sub modules would be skipped in the deploy phase. Unless set too false. Meaning we won't skip them.